### PR TITLE
convert torch no watermark pth to mindspore ckpt

### PR DESCRIPTION
**Pull request by Songyuanwei**

### CODE REVIEW

**Comment by SamitHuang** [comment]
please resolve conflicts

**Review by zhtmike — APPROVED** [review]

File: examples/videocomposer/vc/utils/pt2ms.py
Comment (SamitHuang) [inline examples/videocomposer/vc/utils/pt2ms.py:1]: Please put this conversion script in `videocomposer/tools/.`

File: examples/videocomposer/vc/utils/pt2ms.py [hunk 1]
```diff
@@ -132,3 +133,62 @@ def load_pt_weights_in_model(model, checkpoint_file_pt, state_dict_refiners=None
         dst_path = f"{src_path}.bak"
         print(f"moving {src_path} to {dst_path} ...")
         shutil.move(src_path, dst_path)
+
+    path = "model_weights/non_ema_141000_no_watermark.npy"
```
Comment (SamitHuang) [inline examples/videocomposer/vc/utils/pt2ms.py:137]: Where is the npy from? Try to allow this two-step conversion (pth -> npy -> ckpt) to execute in one command.

Please update readme to give illustartion.

Comment (Songyuanwei) [reply examples/videocomposer/vc/utils/pt2ms.py]: done

File: examples/videocomposer/vc/utils/pt2ms.py
Comment (Songyuanwei) [reply examples/videocomposer/vc/utils/pt2ms.py:1]: done

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**